builder/proxmox/common/config_test.go

```diff
@@ -421,3 +422,131 @@ func TestVMID(t *testing.T) {
 		})
 	}
 }
+
+func TestPCIDeviceMapping(t *testing.T) {
+	testCases := []struct {
+		expectedError   error
+		pciDeviceConfig pciDeviceConfig
+		machine         string
+		vga             vgaConfig
+	}{
+		// Host PCI ID / Mapping
+		{
+			expectedError: fmt.Errorf("either the host or the mapping key must be specified"),
+		},
+		{
+			expectedError: fmt.Errorf("host contains invalid PCI ID"),
+			pciDeviceConfig: pciDeviceConfig{
+				Host: "invalid-pci-id",
+			},
+		},
+		{
+			pciDeviceConfig: pciDeviceConfig{
+				Host: "0000:03:00.0",
+			},
+		},
+		{
+			pciDeviceConfig: pciDeviceConfig{
+				Mapping: "someNic",
+			},
+		},
```

Comment:
Would it be possible to add a test with both `Mapping` and `Host` set to make sure `Prepare` returns an error in this case?